packages/ckeditor5-html-support/tests/manual/features.html

```diff
@@ -0,0 +1,123 @@
+			<p><sup style="color:blue;">Feature superscript</sup></p>
+
+			<p><a style="color:green;" href="https://example.com">Link feature</a></p>
+			<p><a name="anchor" id="anchor">Anchor (name, ID only)</a></p>
+
+			<p><mark class="marker-yellow" data-mark>Mark feature</mark></p>
+
+			<p><span class="text-big text-italic" style="background-color:hsl(60, 75%, 60%);color:hsl(240, 75%, 60%);font-family:'Courier New', Courier, monospace;border:1px solid black">Font feature</span></p>
+
+			<ul style="background:blue;">
+				<li style="color:yellow;">Bulleted List feature</li>
+				<li style="color:yellow;">Bulleted List feature</li>
+				<li style="color:yellow;">Bulleted List feature</li>
+			</ul>
+
+			<ol style="background:blue;">
+				<li style="color:yellow;">Numbered List feature</li>
+				<li style="color:yellow;">Numbered List feature</li>
+				<li style="color:yellow;">Numbered List feature</li>
+			</ol>
+
+			<ul class="todo-list" style="background:blue;">
+				<li style="color:yellow;">
+					<label class="todo-list__label">
+						<input type="checkbox" disabled="disabled">
+						<span class="todo-list__label__description">Todo List feature</span>
+					</label>
+				</li>
+				<li style="color:yellow;">
+					<label class="todo-list__label">
+						<input type="checkbox" disabled="disabled">
+						<span class="todo-list__label__description">Todo List feature</span>
+					</label>
+				</li>
+				<li style="color:yellow;">
+					<label class="todo-list__label">
+						<input type="checkbox" disabled="disabled">
+						<span class="todo-list__label__description">Todo List feature</span>
+					</label>
+				</li>
+			</ul>
+
+			<blockquote style="color:blue;"><p>Blockquote Feature</p></blockquote>
+
+			<figure style="border: 1px solid blue;" class="image">
+				<img src="sample.jpg" style="border: 1px solid red;" alt="bar">
+				<figcaption style="background:yellow;">Caption</figcaption>
+			</figure>
+
+			<pre style="background:blue;"><code style="color:yellow;" class="language-plaintext">Code Block</code></pre>
+
+			<table style="background:blue;">
+				<thead style="background:green;">
+					<tr>
+						<th style="color:red;">h1</th>
+						<th style="color:red;">h2</th>
+						<th style="color:red;">h3</th>
+					</tr>
+				</thead>
+				<tbody style="background:violet;">
+					<tr>
+						<td style="color:yellow;">1.1</td>
+						<td style="color:yellow;">1.2</td>
+						<td style="color:yellow;">1.3</td>
+					</tr>
+					<tr style="color:yellow;">
+						<td>2.1</td>
+						<td>2.2</td>
+						<td>2.3</td>
+					</tr>
+				</tbody>
+			</table>
+
+			<figure class="media" data-foo><oembed data-bar url="https://www.youtube.com/watch?v=ZVv7UMQPEWk"></oembed></figure>
+
+			<div style="border: 1px solid blue" class="raw-html-embed">HTML snippet</div>
+
+			<div style="border: 1px solid blue" class="page-break" style="page-break-after:always;"><span style="display:none;">&nbsp;</span></div>
+		</div>
```

Comment:
We are missing schema definitions for the Page Break and HTML Snippet features. I thought it will require some special integration but I did some quick check and it seems like it may work out of the box. I will check out it next week.